root/user/profile/layout.tt

```diff
@@ -9,15 +9,30 @@
 
 [%- info_links = [
     ['index', simple_link(c.uri_for_action("/user/profile", [ user.name ]), l('Profile'))],
+] -%]
+
+[%- IF c.user.is_account_admin || viewing_own_profile || user.preferences.public_subscriptions -%]
+[%- info_links = [
```

Comment:
Shouldn't this (and below) be using `info_links.push` so that they don't overwrite the previous items?

Reply:
Oops! Fixed in https://github.com/metabrainz/musicbrainz-server/pull/581/commits/2a608a2afd7fe42fa82b5f099607a95950e87e77